.github/workflows/ocaml.yaml

```diff
@@ -9,9 +9,8 @@ jobs:
     strategy:
       matrix:
         include:
-          - ocaml-compiler: "ocaml-base-compiler.5.0.0"
-          - ocaml-compiler: "ocaml-base-compiler.5.1.1"
-            with-doc: true
+          - ocaml-compiler: "ocaml-base-compiler.5.2.0~alpha1"
+            with-doc: false
```

Comment:
the documentation had to be disabled because `odoc` is not compatible with OCaml 5.2 yet (there should be a preview soon enough but it's not there yet)